src/utils/helpers.ts

```diff
@@ -168,28 +209,20 @@ async function getDiscussionUrl(
     );
 
   if (discussions[0]) {
-    const discussion = discussions[0];
-    url = discussion.url;
-
-    let comments = discussion.comments.nodes;
-
-    let latestCommentId: string | number;
-    if (comments?.length) {
-      latestCommentId = getLatestDiscussionCommentId(comments);
-      url += `#discussioncomment-${latestCommentId}`;
-    }
+    return discussions[0];
   }
 
-  return url;
+  return null;
```

Comment:
Is this not the same as just returning `discussions[0]`?

Reply:
Good feedback.  Updated